excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/CategoryParser.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.ccd.definition.store.excel.parser;
+
+import lombok.val;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationError;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationException;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
+import uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception.InvalidImportException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionDataItem;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionSheet;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.ColumnName;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.SheetName;
+import uk.gov.hmcts.ccd.definition.store.excel.validation.CategoryValidator;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CategoryEntity;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeLiteEntity.toCaseTypeLiteEntity;
+
+@Component
+public class CategoryParser {
+
+    private CategoryValidator categoryValidator;
+
+    @Autowired
+    public CategoryParser(CategoryValidator categoryValidator) {
+        this.categoryValidator = categoryValidator;
+    }
+
+    public List<CategoryEntity> parse(Map<String, DefinitionSheet> definitionSheets,
+                                      ParseContext parseContext) {
+        try {
+            val categoriesItems = definitionSheets.get(SheetName.CATEGORY.getName()).getDataItems();
+            categoryValidator.validate(parseContext, categoriesItems);
+            final List<CategoryEntity> newCategoriesEntities = categoriesItems
+                .stream().map(categoryItems -> {
+                    val categoriesEntity = createCategoriesEntity(parseContext, categoryItems);
+                    return categoriesEntity;
+                }).collect(Collectors.toList());
+            return newCategoriesEntities;
```

Comment:
```suggestion
            return categoriesItems
                .stream().map(categoryItems -> createCategoriesEntity(parseContext, categoryItems))
                .collect(Collectors.toList());
```
Can simplify this a little